app/views/articles/_form.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= form_for @article do |f| %>
+
+  <% if @article.errors.any? %>
+    <div id="error_explanation">
+      <h2>
+        <%= pluralize(@article.errors.count, "error") %> prohibited
+        this article from being saved:
+      </h2>
+      <ul>
+        <% @article.errors.full_messages.each do |msg| %>
+          <li><%= msg %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
This format, and the use of tables in the index, make me think this was generated with `rails generate scaffold`, which isn't wrong per se, but I'd encourage you (if you did) to try and avoid using that generator as much as possible, since it's really important to understand how to build this code from scratch